### PR TITLE
add index page for gateway

### PR DESCRIPTION
**Pull request by sophiaso**

### CODE REVIEW

File: initializr-generator/src/main/java/io/spring/initializr/generator/ServiceMetadata.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        // Use relative path for services routed by gateway
+        serviceLinks.add(new ServiceLink(azureServiceBus, "/" + azureServiceBus + "/", false));
+
+        // Cloud Config Server
+        annotationMap.put(cloudConfigServer, Arrays.asList("@SpringBootApplication", "@EnableConfigServer"));
+        importMap.put(cloudConfigServer, Arrays.asList(
+                "org.springframework.boot.autoconfigure.SpringBootApplication",
+                "org.springframework.cloud.config.server.EnableConfigServer"));
+        portMap.put(cloudConfigServer, 8888);
+        serviceLinks.add(new ServiceLink(cloudConfigServer, "http://localhost:8888/application-native.yml", true));
```
Comment (yaweiw) [inline initializr-generator/src/main/java/io/spring/initializr/generator/ServiceMetadata.java:37]: what happens if service is deployed in the cloud? would localhost still work?

File: initializr-generator/src/main/resources/templates/GatewayIndex.tmpl [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="bulma.min.css"/>
+	<title>Gateway Homepage</title>
+</head>
+<body>
```
Comment (yaweiw) [inline initializr-generator/src/main/resources/templates/GatewayIndex.tmpl:7]: Can we add a image in the head of the page? something like this:
https://www.google.com/imgres?imgurl=https%3A%2F%2Fres.infoq.com%2Fpresentations%2Fspring-cloud-gateway%2Fen%2Fslides%2Fsl14-1518709821750.jpg&imgrefurl=https%3A%2F%2Fwww.infoq.com%2Fpresentations%2Fspring-cloud-gateway&docid=JHp8KAAFotD7hM&tbnid=ZE56qk121xqFiM%3A&vet=10ahUKEwi3nrT0tuvaAhUIe7wKHSrEDusQMwg8KAMwAw..i&w=1680&h=945&bih=776&biw=1600&q=spring%20cloud%20gateway&ved=0ahUKEwi3nrT0tuvaAhUIe7wKHSrEDusQMwg8KAMwAw&iact=mrc&uact=8?

**Review by yaweiw — APPROVED** [review]

File: initializr-generator/src/main/resources/templates/GatewayIndex.tmpl (hunk 2 above)
Comment (sophiaso) [reply initializr-generator/src/main/resources/templates/GatewayIndex.tmpl:7]: Styles improved.

File: initializr-generator/src/main/java/io/spring/initializr/generator/ServiceMetadata.java (hunk 1 above)
Comment (sophiaso) [reply initializr-generator/src/main/java/io/spring/initializr/generator/ServiceMetadata.java:37]: Will not work, opened issue #57

**Pull request merged**